docs/software/vpn.md

```diff
@@ -225,4 +226,72 @@ YouTube video tutorial (German language): `Raspberry Pi & PiVPN mit WireGuard: I
 
 <iframe src="https://www.youtube-nocookie.com/embed/yRkdzGmnvA4?rel=0" frameborder="0" allow="fullscreen" width="560" height="315" loading="lazy"></iframe>
 
+## Tailscale
+
+Zero config VPN.
+
+Tailscale is a VPN service that makes the devices and applications you own accessible anywhere in the world, securely and effortlessly. It enables encrypted point-to-point connections using the open source WireGuard protocol, which means only devices on your private network can communicate with each other.
+
+![Tailscale logo](../assets/images/tailscale-logo.svg){: width="242" height="44" loading="lazy"}
+
+=== "Quick start"
+
+    Step 1: Sign up for an account
+
+    [Sign up for a Tailscale account](https://login.tailscale.com/start). Get started with a free personal plan or trial for an organizational plan.
+
+    Tailscale requires a Single Sign-On (SSO) provider, so you’ll need a Google, Microsoft, GitHub, Okta, OneLogin, or other supported SSO identity provider account to begin.
+
+    Step 2: Add a machine to your network
```

Comment:
It's not clear if you're adding this machine, or something else, especially with the line below.

```suggestion
    Step 2: Add this device to your network
```